client/src/components/footer/Footer.css

```diff
@@ -2,7 +2,15 @@
 	width: 100%;
 	background-color: #f1f1f1;
 	display: flex;
+	flex-direction: column;
 	padding: 12px 0;
+	align-items: center;
+}
+
+.footer-top {
+	width: 100%;
+	display: flex;
```

Comment:
Try reducing redundant css properties.
E.g. `footer` is already defined as display: flex, hence the child : `footer-top` will inherit the same.


Also you can try grouping selectors for common properties among various entities.

Reply:
Fixed other redundant CSS properties.
In this particular case, the `footer `is `flex-column` and I need `footer-top` in `flex-row`. So not changing that.
Thx for reviewing